sphinx/source/rpc_api.rst

```diff
@@ -1,6 +1,76 @@
 RPC API
 =======
 
+The available RPC methods vary depending on your TLS connection identity. Some methods are common to all frontends, others are restricted to the member or management frontends, and the app logic is only exposed to users.
+
+The API can also be retrieved from a running service using the listMethods and getSchema methods. For example, using the CCF client application:
+
+.. code-block:: bash
+
+    $ ./client --pretty-print --host 127.99.16.14 --port 36785 --ca networkcert.pem userrpc --req listMethods.json --cert user1_cert.pem --pk user1_privk.pem
```

Comment:
Should we add a link to the section that describes the format of `listMethods.json` and `getSchema.json` as well?